wandb/__init__.pyi

```diff
@@ -969,20 +974,41 @@ def use_artifact(
     Call `download` or `file` on the returned object to get the contents locally.
 
     Args:
-        artifact_or_name: (str or Artifact) An artifact name.
-            May be prefixed with project/ or entity/project/.
-            If no entity is specified in the name, the Run or API setting's entity is used.
-            Valid names can be in the following forms:
-                - name:version
-                - name:alias
-            You can also pass an Artifact object created by calling `wandb.Artifact`
-        type: (str, optional) The type of artifact to use.
-        aliases: (list, optional) Aliases to apply to this artifact
-        use_as: (string, optional) Optional string indicating what purpose the artifact was used with.
-                                   Will be shown in UI.
+        artifact_or_name: The name of the artifact to use. May be prefixed
+            with the name of the project the artifact was logged to
+            ("<entity>" or "<entity>/<project>"). If no
+            entity is specified in the name, the Run or API setting's entity is used.
+            Valid names can be in the following forms
+        - name:version
+        - name:alias
```

Comment:
should this be indented?

Reply:
Lazydocs is a little weird with bullet points. This is how it looks: 

<img width="923" alt="Screenshot 2025-05-13 at 8 04 30 PM" src="https://github.com/user-attachments/assets/786ff6eb-5ea4-4df9-88e8-bd34fb4da2a0" />